src/Curation/Curation.router.spec.ts

```diff
@@ -594,6 +595,115 @@ describe('when handling a request', () => {
               updated_at: expect.any(Date),
             })
           })
+
+          describe('and the collection contains an smart wearable', () => {
+            let mockItem: ItemAttributes
+            let itemSpy: jest.SpyInstance<
+              ReturnType<typeof Item['findByCollectionIds']>
+            >
+
+            beforeEach(() => {
+              mockItem = {
+                ...dbItemMock,
+                contents: {
+                  ...dbItemMock.contents,
+                  'male/game.js': 'hash1',
+                  [VIDEO_PATH]: 'videoHash',
+                },
+                video: 'videoHash',
+              }
+              updateSpy.mockRestore()
+              expectedCuration = {
+                ...expectedCuration,
+                status: CurationStatus.APPROVED,
+              }
+              req = {
+                ...req,
+                body: {
+                  curation: {
+                    status: CurationStatus.APPROVED,
+                  },
+                },
+              } as any
+            })
+
+            describe('and the smart wearable has updated the video', () => {
+              const newVideoHash = 'newVideoHash'
+
+              beforeEach(() => {
+                mockItem = {
+                  ...mockItem,
+                  contents: {
+                    ...mockItem.contents,
+                    [VIDEO_PATH]: newVideoHash,
+                  },
+                  video: '',
+                }
+                itemSpy = jest
+                  .spyOn(Item, 'findByCollectionIds')
+                  .mockResolvedValueOnce([
+                    {
+                      ...mockItem,
+                      total_count: 1,
+                    },
+                  ])
+              })
+
+              it('should resolved with the inserted curation and update the item video', async () => {
+                updateSpy = jest
+                  .spyOn(service, 'updateById')
+                  .mockResolvedValueOnce(expectedCuration)
+                const updateItemSpy = jest
+                  .spyOn(Item, 'upsert')
+                  .mockResolvedValue({
+                    ...mockItem,
+                    video: newVideoHash,
+                  })
```

Comment:
Would you mind setting these two in the `beforeEach`?
The same goes for the other test.